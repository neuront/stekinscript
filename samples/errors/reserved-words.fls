_ + X + NOT +
    RESERVED + WORD + __ + A
