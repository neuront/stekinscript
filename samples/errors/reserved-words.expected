Line: 1
    use reserved word: _
Line: 1
    use reserved word: X
Line: 2
    use reserved word: __
Line: 2
    use reserved word: A
